etc/liblore-copy.sh

```diff
@@ -0,0 +1,451 @@
+LORE_URL='https://lore.kernel.org/'
+CACHE='./cache'
+
+declare -gr SEPARATOR_CHAR='Æ'
+
+# This is a global array that we use to store the list of new patches from a
+# target mailing list. After we parse the data from lore, we will have a list
+# that follows this pattern:
+#
+#  Author, email, version, total patches, patch title, link
+#
+#Note: To separate those elements, we use the variable SEPARATOR_CHAR, which
+#can be a ',' but by default, we use 'Æ'. We used ',' in the example for make
+#it easy to undertand.
+declare -ag list_of_mailinglist_patches
+
+declare -r BASE_PATH_MBOX_DOWNLOAD="$PWD/mbox_download"
```

Comment:
Use `{PWD}`

Reply:
This file is only here as temporary reference, so it will be deleted later.